apps/aecore/src/aec_db.erl

```diff
@@ -167,14 +167,14 @@ write_header(Header) ->
 
 write_header(Header, HasBlock) ->
     {ok, Hash} = aec_headers:hash_header(Header),
-    PrevHash = aec_headers:prev_hash(Header),
+    Height = aec_headers:height(Header),
     ?t(case mnesia:read(aec_headers, Hash) of
            [#aec_headers{value = Header, has_block = OldHasBlock} = H] ->
                NewHasBlock = (HasBlock or OldHasBlock),
                mnesia:write(H#aec_headers{has_block = NewHasBlock});
            [#aec_headers{value = Old}] -> error({header_exist, Header, Old});
            [] -> mnesia:write(#aec_headers{key = Hash, value = Header,
-                                           prev_hash = PrevHash,
+                                           height = Height,
```

Comment:
It looks like this change in DB schema is so small that a new node may manage to start with an old DB. We [document to the user not to reuse persisted blockchain ("Please ensure that you do not reuse a persisted blockchain produced by the previous releases "v0.8.x".")](https://github.com/aeternity/epoch/blob/4795b2c820e1f44c23a92cb62ef5c85ec47cbd9e/docs/release-notes/RELEASE-NOTES-0.9.0.md#about-this-release) so it is fine. (We did not [teach node to detect incompatible DB schema](https://www.pivotaltracker.com/story/show/155656073)).

Reply:
The schema is altered because of the new field. Will Mnesia really start up with the persisted DB?

But, I guess the node will not shut down as it should.

Reply:
> Will Mnesia really start up with the persisted DB?

I did not test.